libkbfs/folder_block_ops.go

```diff
@@ -694,43 +675,43 @@ func (fbo *folderBlockOps) GetDirtyEntry(
 	return fbo.getDirtyEntryLocked(ctx, lState, md, file)
 }
 
+func (fbo *folderBlockOps) getOrCreateDirtyFileLocked(lState *lockState,
```

Comment:
Sure, I can do that.  But note that .kbfs_status within a TLF already lists the set of dirty files via another mechanism in folderBranchOps. We can probably eventually get rid of that other way, though.